tests/test_report_manager.py

```diff
@@ -254,9 +252,10 @@ def subtest_build_summary_table(self, mode, cpu_only):
             self.assertGreaterEqual(current_row[throughput_index],
                                     next_row[throughput_index])
 
-    def test_build_detailed_info(self):
-        for cpu_only in [True, False]:
-            self._subtest_build_detailed_info(cpu_only)
+    # FIXME: You can't add results if the config is REPORT
```

Comment:
@tgerdesnv  - This unit test is problematic and I can't think of a good way of easily fixing it. The issue is that you can't add new measurements if the report subcommand is used (as we don't have any run_comparators)

Reply:
We need to be able to test this text creation. There must be a way to do it if the live code is working.

Reply:
I fixed the unit test